Corona-Warn-App/src/main/java/de/rki/coronawarnapp/contactdiary/ui/day/tabs/person/ContactDiaryPersonListFragment.kt

```diff
@@ -49,7 +48,11 @@ class ContactDiaryPersonListFragment : Fragment(R.layout.contact_diary_person_li
         viewModel.uiList.observe2(this) {
             personListAdapter.update(it)
 
-            binding.contactDiaryPersonListNoItemsGroup.setInvisible(it.isNotEmpty())
+            if (it.isNotEmpty()) {
+                binding.contactDiaryPersonListNoItemsGroup.visibility = View.GONE
+            } else {
+                binding.contactDiaryPersonListNoItemsGroup.visibility = View.VISIBLE
+            }
```

Comment:
same here as in https://github.com/corona-warn-app/cwa-app-android/pull/1983/files#r547947203